addons/gamification/data/gamification_karma_tracking_demo.xml

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <!--base.user_demo-->
+        <record id="tracking_03" model="gamification.karma.tracking">
```

Comment:
Maybe change your ids to something more meaningful.

Something like targeted_user_date would be good.
eg: "user_demo_2016_01_01"

It will help other modules referencing those demo when necessary and also won't require to change all ids if you want to add a tracking between "tracking_01" and "tracking_02"